examples/example01.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+include (__DIR__ . "/../vendor/autoload.php");
```

Comment:
Ja bym ten plik wykluczył z packagista. `.gitattribute export-ignore` to słowo klucz

Reply:
Done